cli.ts

```diff
@@ -6,8 +6,9 @@ const args = require("minimist")(process.argv.slice(2), {
   alias: {
     cid: "client_id",
     tid: "tenant_id",
-    ci: "continuous_integration_variable"
+    ci: "continuous_integration_variable",
+    pbp: "project_base_path"
```

Comment:
This will be required parameter after this change?

Reply:
Nope - just optional like the others. It will use the current working directory if not specified, which is what it was doing before